src/components/Hr/Hr.tsx

```diff
@@ -24,10 +24,12 @@ export interface Props {
  * Primary UI component for user interaction
  */
 export const Hr: React.FC<Props> = ({ className, size, variant, ...other }) => {
-  const componentClassName = clsx(styles['hr'], className, {
-    [styles['hr--lg']]: size === 'lg',
-    [styles['hr--brand']]: variant === 'brand',
-  });
+  const componentClassName = clsx(
+    styles['hr'],
+    className,
+    size === 'lg' && styles['hr--lg'],
+    size === 'brand' && styles['hr--brand'],
```

Comment:
I think this should be
```suggestion
    variant === 'brand' && styles['hr--brand'],
```
and the prop type doesn't need to change because there's already a `variant` prop